{
  "comments": [
    {
      "key": {
        "uuid": "34c3ee5c_83f72877",
        "filename": "/COMMIT_MSG",
        "patchSetId": 1
      },
      "lineNbr": 7,
      "author": {
        "id": 1041833
      },
      "writtenOn": "2016-01-26T04:14:53Z",
      "side": 1,
      "message": "Typo",
      "range": {
        "startLine": 7,
        "startChar": 21,
        "endLine": 7,
        "endChar": 26
      },
      "revId": "ca7c6ace3ba3040dc8f3cd3b621cc28378cb00f6",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": false
    },
    {
      "key": {
        "uuid": "74c9e678_caa9339d",
        "filename": "runtime/oat_file_assistant.cc",
        "patchSetId": 1
      },
      "lineNbr": 1030,
      "author": {
        "id": 1041833
      },
      "writtenOn": "2016-01-26T04:14:53Z",
      "side": 1,
      "message": "I think it would be interesting to note a failure if the file existed...",
      "range": {
        "startLine": 1030,
        "startChar": 4,
        "endLine": 1030,
        "endChar": 87
      },
      "revId": "ca7c6ace3ba3040dc8f3cd3b621cc28378cb00f6",
      "serverId": "85c56323-6fa9-3386-8a01-6480fb634889",
      "unresolved": false
    }
  ]
}